composer.json

```diff
@@ -44,14 +44,17 @@
         "2amigos/yii2-selectize-widget": "^1.1",
         "yiisoft/yii2-authclient": "^2.1",
         "yiisoft/yii2-httpclient": "^2.0",
-        "yiisoft/yii2-bootstrap": "^2.0",
-        "yiisoft/yii2-swiftmailer": "^2.0"
+        "yiisoft/yii2-bootstrap": "^2.0"
     },
     "suggest": {
+        "yiisoft/yii2-symfonymailer": "A mailer driver is needed to send e-mails. Older versions use abandoned Swiftmailer which can be replaced with symfonymailer",
         "2amigos/2fa-library": "Needed if you want to enable 2 Factor Authentication. Require version ^1.0",
         "2amigos/qrcode-library": "Needed if you want to enable 2FA with QR Code generation. Require version ^1.1"
     },
     "require-dev": {
+        "php": ">=7.4",
```

Comment:
There's a `require` for PHP 5.5, I guess that's to be upgraded

Reply:
Currently it runs fine on php5.6 as required. The 7.4 is optional and required only for dev because the Yii's own TestMailer wants to use the SwiftMailer/message by default. Until Yii is not also cleared from SwiftMailer we need to use some external mailer for tests (thus the SymphonyMailer & php7.4). So in principal it could also stay like that, Just tests are not run for older versions.

But, having said that, eventually old versions also need to be killed. So I am thumbs up for moving up the minimum required php to 7.4. Also then a major version tag would be a good idea to indicate breaking changes.

But, within this PR, nothing should be broken unless swiftmailer is used and ONLY required by usuario, in this scenario one would end up not having the dependency and not being able to email.

Reply:
Ok, fine. We can bump the requirement with a new major tag.

I wonder where can we *save* this requirement change, so I can remember when tagging the release :D